components/cpm/Performance.vue

```diff
@@ -1,54 +1,119 @@
 <template>
   <div class="w-full h-full">
-    <span class="space-x-3 p-3">
-      <CpmDropdown v-model="selectedVariable" :options="variables" alttext="Choose a variable." />
-      <CpmDropdown v-model="selectedSeason" :options="seasons" alttext="Select a season." />
-      <CpmDropdown v-model="selectedModel" :options="filterModels" alttext="Choose a model/group" />
+    <span class="flex flex-wrap space-x-3 p-3">
+      <div>
+        <multiselect
+          v-model="selectedVariable"
+          :options="variables"
+          :searchable="false"
+          :allow-empty="false"
+          :close-on-select="true"
+          :show-labels="false"
+          label="name"
+          deselect-label=" "
+          placeholder="Choose a variable."
+        />
+      </div>
+      <div>
+        <multiselect
+          v-model="selectedSeason"
+          :options="seasons"
+          :searchable="false"
+          :allow-empty="false"
+          :close-on-select="true"
+          :show-labels="false"
+          label="name"
+          deselect-label=" "
+          placeholder="Select a season."
+        />
+      </div>
+      <div>
+        <multiselect
+          v-model="selectedModel"
+          :options="filterModels"
+          :searchable="true"
+          :allow-empty="false"
+          :close-on-select="true"
+          :show-labels="false"
+          label="name"
+          deselect-label=" "
+          placeholder="Choose a model/group."
+        />
+      </div>
+      <div>
+        <multiselect
+          v-model="selectedSystem"
+          :options="modellingSystems"
+          :multiple="true"
+          :close-on-select="false"
+          :clear-on-select="false"
+          :preserve-search="true"
+          :show-labels="true"
+          placeholder="Choose modelling system(s)"
+          label="name"
+          track-by="name"
+          :preselect-first="true"
+        >
+          <template slot="selection" slot-scope="{ values, isOpen }"><span v-if="values.length &amp;&amp; !isOpen" class="multiselect__single">{{ values.length }} modelling system(s) selected</span></template>
```

Comment:
Modelling systems is not clear and also not really accurate. I think I'd prefer either:

- Projects (CMIP, CORDEX, CPM)  (this is accurate)
- Type (global, regional, convection-permitting)   (this holds the middle)
- Resolution (low, medium, high)   (this is easier to understand)

Reply:
Thanks for your suggestions. I prefer the accurate one, the Projects (CMIP, CORDEX, CPM). They are implemented now.